src/frontend/scss/objects/_course_glimpses.scss

```diff
@@ -150,6 +150,14 @@ $course-glimpse-content-padding-sides: 0.7rem !default;
     position: relative;
   }
 
+  &__code {
+    font-size: 0.7rem;
+    color: r-theme-val(course-glimpse, content-color);
+    line-height: 1rem;
+    min-height: 1rem;
+    margin-bottom: 0;
```

Comment:
Sort line ascending.

I don't understand the use of `min-height`, `line-height` should be sufficient here?

Reply:
Apparently `line-height` doesn't keep the line height when the code is null that's why I had to add a `min-height` property as I don't want the title to go up in this case.

Reply:
Ah ok! From my point of view, if `course.code` is null, the DOM should not contain the node `course-glimpse__code`.

So may be we could add a `margin-top: 1rem` to `course-glimpse*__title`
Then a `margin-bottom: -1rem` to `course-glimpse*__code. In this title will have a margin-top to 1rem only if there is no `course.code` above.

Reply:
I asked myself this question and in view of schema.org I thought the right way to do it is to keep the markup and give the information that the code is null.

Reply:
Oh ok, I see your point. Let's this node empty ! 👍